### PR TITLE
Port Challenge 3 Snake Game to new site

### PR DESCRIPTION
**Pull request by lobertrand**
- I created thumbnails for most of the passenger showcases.
- I used scrapped data by @skaser85 https://github.com/skaser85/thecodingtrain_website_data/tree/main/challenges/3-the-snake-game/showcase (thanks)
- I deleted showcases 7, 17 et 22 because their links where all broken
- I deleted showcase 3 because it was a duplicate of showcase 8

### CODE REVIEW

**Comment by shiffman** [comment]
Great! I'd suggest merging with main and checking out some of the latest challenge pages as things have been slowly evolving and improving!

File: content/videos/challenges/3-snake-game/index.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "title": "The Snake Game",
+  "description": "In this coding challenge, I attempt to code a p5.js version of the Snake Game.",
+  "videoNumber": "3",
+  "videoId": "AaGK-fj-BAM",
+  "date": "2016-04-20",
+  "languages": [],
```
Comment (klinegareth) [inline content/videos/challenges/3-snake-game/index.json:7]:
```suggestion
  "languages": ["p5.js", "JavaScript"],
```

File: content/videos/challenges/3-snake-game/showcase/contribution11.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "title": "Python Edition",
+  "url": "",
```
Comment (klinegareth) [inline content/videos/challenges/3-snake-game/showcase/contribution11.json:3]: Curious what this looks like on the site, might want to put the GitHub link here instead

Comment (lobertrand) [reply content/videos/challenges/3-snake-game/showcase/contribution11.json]: You're right, definitely need to fill in that url or else it redirects to the same page in another tab :)

**Comment by shiffman** [comment]
Just checked this one and it looks great! Merging, thank you for your patience @lobertrand!

**Pull request merged**